flang/lib/Lower/ConvertVariable.cpp

```diff
@@ -1063,9 +1061,6 @@ void Fortran::lower::mapSymbolAttributes(
     }
   };
 
-  if (isHostAssoc)
-    TODO(loc, "host associated");
```

Comment:
Why are you removing this TODO ? internal procedure are not implemented yet, so it looked legit to me.

Reply:
It seems misplaced. This code is determining the properties associated with a variable, which really have nothing to do with whether the variable is host associated or not.